### PR TITLE
install_guide: fix disclaimer

### PR DESCRIPTION
**Pull request by hasbro17**
[skip ci]
fix for #1663

### CODE REVIEW

File: doc/user/install_guide.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ kubectl delete clusterrolebinding etcd-operator
 ```
 
 ## Installation via Helm
-**Disclaimer:** The following Helm chart is an external project not maintained by the etcd-operator maintainers. It may not be up to date and so is not the recommended way to install the etcd-operator.
+**Disclaimer:** The following Helm chart is an external project not maintained by the etcd-operator maintainers, so it may not be up to date.
```
Comment (fanminshi) [inline doc/user/install_guide.md]: prefer `;` to connect two closely full sentence together.
`maintainers; so it` or `maintainers; the chart` 

Just a suggestion.

**Comment by hasbro17** [comment]
fixed.

**Comment by xiang90** [comment]
lgtm

**Comment by fanminshi** [comment]
lgtm

**Pull request merged**